qlora.py

```diff
@@ -463,14 +471,14 @@ def local_dataset(dataset_name):
     if dataset_name.endswith('.json'):
         full_dataset = Dataset.from_json(path_or_paths=dataset_name)
     elif dataset_name.endswith('.jsonl'):
-        full_dataset = Dataset.from_json(filename=dataset_name, format='jsonlines')
+        full_dataset = Dataset.from_pandas(pd.read_json(dataset_name, lines=True))
```

Comment:
here is the code suggestion to revert it, if you or a maintainer would like to commit it :)
```suggestion
        full_dataset = Dataset.from_json(filename=dataset_name, format='jsonlines')
```

Reply:
Thank you! @lhoestq